### PR TITLE
Add basic UI skeleton for stall page

### PR DESCRIPTION
**Pull request by niqiukun**
Screenshot:

image

### CODE REVIEW

**Review by Kalsyc — APPROVED** [review]
LGTM

File: src/pages/SearchPage/SearchPage.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ const SearchPage: React.FunctionComponent = () => {
     getAllStalls().then(response => {
       setStalls(response.data);
     })
-  })
+  }, []);
```
Comment (Kalsyc) [inline src/pages/SearchPage/SearchPage.tsx:20]: Yup, I forgot lmao

**Comment by Kalsyc** [comment]
I'll do the linking of the stalls to the page after merging this in my own branch

File: src/pages/StallPage/StallPage.tsx [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Button, Grid, GridColumn, Rating } from 'semantic-ui-react';
+import FoodGrid from '../../components/FoodGrid/FoodGrid';
+import SearchHeader from '../../components/SearchHeader/SearchHeader';
+import { getStall } from '../../services/stall';
+import Stall from '../../types/Stall';
+import './StallPage.css';
+
+const SearchPage: React.FunctionComponent = () => {
```
Comment (Kalsyc) [inline src/pages/StallPage/StallPage.tsx:10]: `SearchPage` lol

Comment (niqiukun) [reply src/pages/StallPage/StallPage.tsx:10]: Oh oops can help me fix this? Sorry!

**Pull request merged**